modules/odo-default-config-changes.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-migration-guide.adoc
+
+:_content-type: CONCEPT
+[id="odo-default-config-changes_{context}"]
+
+= Changes to default configurations
+
+== Ephemeral storage
+
+By default, `odo` v2 uses _ephemeral storage_ for the components it creates. `odo` v3 now uses the underlying `PersistentVolume` storage that is configured for users. If you want to continue using ephemeral storage for `odo` components, change the default configuration with the `odo preference` command:
```

Comment:
Is there a benefit to keeping with ephemeral storage? if it's simply because users want to keep doing things as they know how, then leave the content as it is. 

If there is a benefit, it would be helpful to state it. Something like:

`By default, `odo` v2 uses _ephemeral storage_ for the components it creates. `odo` v3 now uses the underlying `PersistentVolume` storage that is configured for users. (state the benefit of keeping with ephemeral storage). If you want to continue using ephemeral storage for `odo` components, change the default configuration with the `odo preference` command:`

Reply:
I think if you have persistent storage available on the cluster, there is no reason to use ephemeral